Framework/ICat/src/CatalogDownloadDataFiles.cpp

```diff
@@ -169,6 +169,8 @@ std::string CatalogDownloadDataFiles::doDownloadandSavetoLocalDrive(
     Poco::Net::SecureStreamSocket *socket =
         new Poco::Net::SecureStreamSocket(context);
     Poco::Net::HTTPSClientSession session(*socket);
```

Comment:
The [Poco documentation](https://github.com/pocoproject/poco/blob/develop/NetSSL_OpenSSL/include/Poco/Net/HTTPSClientSession.h#L90) is wrong. There nothing in the destructor!
https://github.com/pocoproject/poco/blob/develop/NetSSL_Win/src/HTTPSClientSession.cpp#L110
https://github.com/pocoproject/poco/blob/develop/NetSSL_OpenSSL/src/HTTPSClientSession.cpp#L110